script.js

```diff
@@ -0,0 +1,26 @@
+function closeMenu() {
+  document.querySelector('.menu-bars').classList.remove('hide');
+  document.querySelector('#desktop-nav').classList.remove('show');
+  document.querySelector('#desktop-nav ul').classList.remove('show');
+  document.querySelector('main').classList.remove('hide');
+  document.querySelector('footer').classList.remove('hide');
+  document.querySelector('h1').classList.remove('hide');
+  document.querySelector('header').classList.remove('menu-visible');
+  document.querySelector('.close-btn').classList.add('hide');
+}
+
+function openMenu() {
+  document.querySelector('.menu-bars').classList.add('hide');
+  document.querySelector('#desktop-nav').classList.add('show');
+  document.querySelector('#desktop-nav ul').classList.add('show');
+  document.querySelector('main').classList.add('hide');
+  document.querySelector('footer').classList.add('hide');
+  document.querySelector('h1').classList.add('hide');
+  document.querySelector('header').classList.add('menu-visible');
+  document.querySelector('.close-btn').classList.remove('hide');
+}
```

Comment:
- [ ] I think you should consider writing a shorter and DRY code to toggle the mobile menu. You can achieve the opening and closing of the mobile menu with a single functionality. Writing simple and DRY code is very important in maintaining code quality. You can consider this example and build on it: 

```
function toggleMenu() {
  document.querySelectorAll('.menu-bar').forEach((element) => element.addEventListener('click', () => {
    document.querySelector('.mobile-menu').classList.toggle('active');
  }));
}

toggleMenu();
```